### PR TITLE
[Fix-5830][Improvement][UI] Improve the selection style in dag edit dialog

### LINKED ISSUE #5830: [Improvement][Web] Improve the selection style in dag edit dialog
*For better global communication, please give priority to using English description, thx! *

**Describe the question**
When we want to select all or part of the text in the input box, because the selected background color is too similar to the original background color, users often think that the text cannot be selected

db6deb90d479ae54c0a0f89a0c8a1c838bc149d4

**Which version of DolphinScheduler:**
latest dev branch

**Describe alternatives you've considered**
A clear and concise description of any alternative improvement solutions you've considered.

### PR DESCRIPTION
**Pull request by echohlne**
## Purpose of the pull request
Fix:#5830
Close #5830 #5352

## Brief change log
dag.scss set a global selection:
```css
.dag-model {
  background: url("../img/dag_bg.png");
  height: calc(100vh - 100px);

  ::selection {
    background: transparent;
  }

  ::-moz-selection {
    background: transparent;
  }

  ::-webkit-selection {
    background: transparent;
  }
# ...
```

I try to set a new background binds el-drawer. 
udp.vue also updates like this above.
## Verify this pull request

Manually verified the change by testing locally.

8b6d0a8d215c34fa53b563ea023dfdf26e0dad3a

image

### CODE REVIEW

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue [hunk 1]
```diff
@@ -893,4 +893,22 @@
...
     padding: 8px 6px;
   }
+
+  .el-drawer__body {
+    ::selection {
+      background: #409EFF;
+      color: white;
+    }
+
+    ::-moz-selection {
```
Comment (chengshiwen) [inline dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue]: Please remove `::-moz-selection` and `::-webkit-selection`, `postcss-loader` and `autoprefixer` will add the prefixes automatically when building the ui

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/udp/udp.vue [hunk 2]
```diff
@@ -251,6 +251,19 @@
       max-height: 600px;
       overflow-y: scroll;
       padding:0 20px;
+
+      ::selection {
+        background: #409EFF ;
+        color: white;
+      }
+      ::-moz-selection {
```
Comment (chengshiwen) [inline dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/udp/udp.vue]: Please remove `::-moz-selection` and `::-webkit-selection`

**Review by chengshiwen — CHANGES_REQUESTED** [review]

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/udp/udp.vue (hunk 2 above)
Comment (echohlne) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/udp/udp.vue]: Thanks @chengshiwen, done.

**Review by chengshiwen — APPROVED** [review]
+1

**Comment by echohlne** [comment]
ping
@zhuangchong

**Pull request merged**